Python/Instagram-Liker-Bot/Instagram_Liker_Bot.py

```diff
@@ -0,0 +1,48 @@
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.common.keys import Keys
+from getpass import getpass
+import time
+
+chrome=webdriver.Chrome(ChromeDriverManager().install())
```

Comment:
CAn we have a more modular approach for this? Make a function and call it from a driver?